pkg/cloudprovider/aws/capacity.go

```diff
@@ -112,3 +149,20 @@ func (c *Capacity) GetOperatingSystems(ctx context.Context) ([]string, error) {
 		v1alpha1.OperatingSystemLinux,
 	}, nil
 }
+
+func (c *Capacity) ValidateLabels(labels map[string]string) error {
```

Comment:
Can we just have the method be Validate() and we can validate arbitrary parts of the spec?

Reply:
Can we break this validator into another file, too?

Reply:
yeah, I think that's similar to what Jacob proposed.

Reply:
Maybe I'm taking this too far, but are you suggesting getting rid of the validators in `webhooks/provisioning/validator.go`? And instead have the new cloudprovider `Validate()` do those validations as well?

Reply:
I think we probably want both.